cmd/signozschemamigrator/schema_migrator/column_operations_test.go

```diff
@@ -0,0 +1,440 @@
+			op:   AlterTableModifyColumn{Database: "db", Table: "table", Column: Column{Name: "col", Default: "'127.0.0.1'"}}.OnCluster("cluster"),
+			want: "ALTER TABLE db.table ON CLUSTER cluster MODIFY COLUMN IF EXISTS col DEFAULT '127.0.0.1'",
+		},
+		{
+			name: "modify-column-compression-codec",
+			op:   AlterTableModifyColumn{Database: "db", Table: "table", Column: Column{Name: "col", Codec: "ZSTD"}}.OnCluster("cluster"),
+			want: "ALTER TABLE db.table ON CLUSTER cluster MODIFY COLUMN IF EXISTS col CODEC(ZSTD)",
+		},
+		{
+			name: "modify-column-ttl",
+			op:   AlterTableModifyColumn{Database: "db", Table: "table", Column: Column{Name: "col", TTL: "d + INTERVAL 1 DAY"}}.OnCluster("cluster"),
+			want: "ALTER TABLE db.table ON CLUSTER cluster MODIFY COLUMN IF EXISTS col TTL d + INTERVAL 1 DAY",
+		},
+		{
+			name: "modify-column-settings",
+			op:   AlterTableModifyColumn{Database: "db", Table: "table", Column: Column{Name: "col", Settings: ColumnSettings{ColumnSetting{Name: "min_compress_block_size", Value: "16777216"}}}}.OnCluster("cluster"),
+			want: "ALTER TABLE db.table ON CLUSTER cluster MODIFY COLUMN IF EXISTS col SETTINGS min_compress_block_size = 16777216",
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			require.Equal(t, tc.want, tc.op.ToSQL())
+		})
+	}
+}
+
+func TestAlterTableModifyColumnRemove(t *testing.T) {
+	testCases := []struct {
+		name string
+		op   Operation
+		want string
+	}{
+		{
+			name: "remove-column-property-ttl",
+			op:   AlterTableModifyColumnRemove{Database: "db", Table: "table", Column: Column{Name: "col"}, Property: ColumnPropertyTTL}.OnCluster("cluster"),
+			want: "ALTER TABLE db.table ON CLUSTER cluster MODIFY COLUMN IF EXISTS col REMOVE TTL",
+		},
+		{
+			name: "remove-column-property-codec",
+			op:   AlterTableModifyColumnRemove{Database: "db", Table: "table", Column: Column{Name: "col"}, Property: ColumnPropertyCodec}.OnCluster("cluster"),
+			want: "ALTER TABLE db.table ON CLUSTER cluster MODIFY COLUMN IF EXISTS col REMOVE CODEC",
+		},
+		{
+			name: "remove-column-property-default",
+			op:   AlterTableModifyColumnRemove{Database: "db", Table: "table", Column: Column{Name: "col"}, Property: ColumnPropertyDefault}.OnCluster("cluster"),
+			want: "ALTER TABLE db.table ON CLUSTER cluster MODIFY COLUMN IF EXISTS col REMOVE DEFAULT",
+		},
+		{
+			name: "remove-column-property-settings",
+			op:   AlterTableModifyColumnRemove{Database: "db", Table: "table", Column: Column{Name: "col"}, Property: ColumnPropertySettings}.OnCluster("cluster"),
+			want: "ALTER TABLE db.table ON CLUSTER cluster MODIFY COLUMN IF EXISTS col REMOVE SETTINGS",
+		},
+		{
+			name: "remove-column-property-materialized",
+			op:   AlterTableModifyColumnRemove{Database: "db", Table: "table", Column: Column{Name: "col"}, Property: ColumnPropertyMaterialized}.OnCluster("cluster"),
+			want: "ALTER TABLE db.table ON CLUSTER cluster MODIFY COLUMN IF EXISTS col REMOVE MATERIALIZED",
```

Comment:
Doesn't something like this exisits ?

Reply:
Yes, It will remove the materialized property of the column if it exists.

Reply:
Interesting, so it will revert it to a normal column is it?

Reply:
Yes